pandas/tests/sparse/test_series.py

```diff
@@ -1360,4 +1360,4 @@ def test_numpy_func_call(self):
                  'argmin', 'argmax', 'min', 'max']
         for func in funcs:
             for series in ('bseries', 'zbseries'):
-                getattr(np, func)(getattr(self, series))
+                getattr(np, func)(getattr(self, series)._values)
```

Comment:
Hmm...you do raise a good point.  Perhaps we should then add a `tm.assert_produces_warning(...)`, which conditions the warning class to be `None` or `FutureWarning` depending on which method is being tested.